### PR TITLE
basic implementation of multiple audio testing done

### PR DESCRIPTION
**Pull request by Anurodhyadav**

### CODE REVIEW

File: src/views/MultiTest.vue [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+                        const myData = {
+                          id: myDoc.id,
+                          testNo: myDoc.data().test.length,
+                          test: myDoc.data().test,
+                        };
+                        this.myUserId = myData.id;
+                        this.testNum = myData.testNo;
+                        this.testPush = myData.test;
+                        this.testPush.push(this.documentID);
+                        this.avgJsdiff = (this.avgJsdiff + parseInt(this.jsDiff, 10)) / this.count;
```
Comment (Ariesleo) [inline src/views/MultiTest.vue:227]: parseInt ko thauma math.Int nai use garda k farak hunxa

File: src/views/MultiTest.vue [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+      .where('user', '==', this.current_User_email)
+      .get()
+      .then((mySnapshot) => {
+        mySnapshot.forEach((Doc) => {
+          const thisData = {
+            jsDiff: Doc.data().jsDiff,
+            grammaraccuracy: Doc.data().grammaraccuracy,
+          };
+          this.jsdiffNet.push(parseInt(thisData.jsDiff, 10));
+          this.grammarAccuracynet.push(parseInt(thisData.grammaraccuracy, 10));
```
Comment (Ariesleo) [inline src/views/MultiTest.vue:377]: we are using Math.Int and parseInt as well. so will it make difference if we use the any one in all the places

**Review by Ariesleo — APPROVED** [review]

File: src/views/MultiTest.vue [hunk 3]
```diff
@@ -0,0 +1,441 @@
+<template>
```
Comment (shakyaprasen) [inline src/views/MultiTest.vue:1]: this looks the same as the test.vue component, any issues in using the same component for these two use cases? Just seems like a lot of duplicated code.

File: src/views/Result.vue [hunk 4]
```diff
@@ -92,7 +92,41 @@
             </vs-td>
           </vs-tr>
         </template>
-      </vs-table>
+      </vs-table> -->
+      <vs-row v-for="(i, j) in this.transInfo" :key="j" vs-justify="center">
```
Comment (shakyaprasen) [inline src/views/Result.vue:96]: please give a better variable name instead of i and j

**Pull request merged**